eclair-core/src/main/scala/fr/acinq/eclair/blockchain/fee/FeeProvider.scala

```diff
@@ -52,6 +53,7 @@ case class FeeratePerKB(feerate: Satoshi) extends Ordered[FeeratePerKB] {
 object FeeratePerKB {
   // @formatter:off
   def apply(feeratePerByte: FeeratePerByte): FeeratePerKB = FeeratePerKB(feeratePerByte.feerate * 1000)
+  // feerate-per-kw should be feerate-per-kvb / 4
```

Comment:
When the code is trivial like here, this comment is really unnecessary, can you remove it?

Reply:
OK. Removed it.